app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/componentHealthCheckControls.jsx

```diff
@@ -106,15 +106,27 @@ export class ComponentHealthCheckControls extends Component {
   formatFilterValue = (value) => value && value[0];
   parseFilterValue = (value) => value && [value];
 
-  renderAttributesFieldArray = ({ fields, fieldValidator }) => (
-    <AttributesFieldArrayControl
-      fields={fields}
-      fieldValidator={fieldValidator}
-      maxAttributesAmount={MAX_ATTRIBUTES_AMOUNT}
-      showRemainingLevels
-      url={this.props.itemAttributeKeysAllSearch}
-    />
-  );
+  renderAttributesFieldArray = ({ fields, fieldValidator }) => {
+    const {
+      activeProject,
+      widgetSettings: { contentParameters, filters },
+    } = this.props;
+    const filterId = filters && filters[0].value;
+    const isLatest =
+      (contentParameters && contentParameters.widgetOptions.latest) ||
+      MODES_VALUES[CHART_MODES.ALL_LAUNCHES];
+    const launchesLimit = DEFAULT_LAUNCHES_LIMIT;
```

Comment:
I think it's unnecessary assignment. You can use constant `DEFAULT_LAUNCHES_LIMIT` where needed